### DISCUSSION TITLE
Get list of images per topic for multimodal topic modeling and save representative images

### DISCUSSION THREAD

**Discussion by nabsiddiqui** [MaartenGr/BERTopic#2235] [Q&A]
Hello,

I have been using BERTopic on some images that I have (no captions) for clustering. I am following the information listed here: https://maartengr.github.io/BERTopic/getting_started/multimodal/multimodal.html#images-only. At the end, the output has the keywords and some representative images. I was wondering if it is possible to get a list of the original image paths for each topic? Preferably ordered at the level of representation. I know all this is available to the model at some point, but I don't know how to get the information out.

**Comment by MaartenGr** [comment]
You can get the assignment of topics to images when accessing `topic_model.topics_`. That list is sorted in the same order as the input images.
